webapp/cypress/integration/login.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Login scenarios
+
+  Background: I am on the "Register" Page
+    Given I am on the site homepage
+    When I click on "Login" button on the "Home" Page
+    Then I am redirected to "Login" Page
+
+  Scenario: I am logged in when I enter correct email and password credentials
+    Given I enter "admin@admin.pl" into "Email" field on the "Login" Page
+    And I enter "admin" into "Password" field on the "Login" Page
+    When I click on "Login" button on the "Login" Page
+    Then I am redirected to "History" Page of "admin@admin.pl" User
+    And Alert "Login successful" is displayed on "History" Page of "admin@admin.pl" User
+
+  Scenario: Alert is displayed when I did not enter email and password credentials
+    When I click on "Login" button on the "Login" Page
```

Comment:
Wszystko co jest w apostrofach powinno być parametrem. Tak naprawdę do nazwa strony jest nam niepotrzebna, więc można ja wszędzie wywalić. Zredukuje nam to trochę ilość testów w plikach *.js

Reply:
poprawione